Dockerfile-asterisk

```diff
@@ -0,0 +1,4 @@
+FROM wazoplatform/asterisk
+
+RUN true \
+    && apt-get update && apt-get install -y asterisk-sounds-main
```

Comment:
You can add `update -qq`.
You can also add `wazo-sounds-main` and `asterisk-sounds-wav-en-us`

Reply:
* I don't have reason to hide this information
* I would add only the minimal working package for my needs
  * The scope was not to add default system sound file

At the end, these Dockerfile should not exist and are only a workaround until we migrate all these logic in each repo

Reply:
Hehe agreed the scope was not explicitly said... sorry :sweat_smile: